src/main/java/games/strategy/triplea/util/AiBonusIncomeUtils.java

```diff
@@ -0,0 +1,38 @@
+package games.strategy.triplea.util;
```

Comment:
So wasn't really sure where to put this but definitely don't want it in the AI packages. The code in the AI packages is supposed to be just AI implementation code. This code is really related to game properties and delegates. Usually I'd just leave it in whatever delegate was using it but since 2 delegates call to it, I put it in a utility class.